UI/src/app/home/home.component.html

```diff
@@ -1,16 +1,18 @@
 <p>Text to Plot.</p>
 <hr>
 <div>
-  <div id="selectBox">
-    <select  class="form-control"
-             [(ngModel)]="dataset">
-      <option *ngFor="let x of datasets"
-              [ngValue]="x"
-              [selected]="x.id === 0">
-        {{ x.name }}
-      </option>
+  <div>
+    <div id="selectBox">
+    <select  class="form-control" [(ngModel)]="dataset">
+      <option *ngFor="let x of datasets" [ngValue]="x" [selected]="x.id === 0">{{ x.name }}</option>
     </select>
-  </div>
+    </div>
+    <app-file-upload #fileUpload
+                     id="file-upload"
+                     [dataList]="datasets">
```

Comment:
does this work both ways?
did someone test, if an uploaded dataset also shows up in the drop down list?

Reply:
Yes. It works. If file upload was successful it will be displayed in selection list